miqi/agent/tools/graph_render.py

```
@@ -0,0 +1,814 @@
+                    exists = candidate.is_file()
+                if exists:
+                    targets.append((candidate, name))
+            if not targets:
+                raise GraphDataError(f"目录 {resolved} 中未找到 {GRAPH_FILE_NAMES}")
+            return targets
+        if is_file:
+            return [(resolved, resolved.name)]
+        raise GraphDataError(f"路径不存在: {resolved}")
+
+    async def execute(
+        self, path: str, format: str = "svg", out_dir: str | None = None, **kwargs: Any
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)